static/hello-world/src/App.tsx

```diff
@@ -1,18 +1,18 @@
 // @ts-nocheck
 import React, { useEffect, useState, useRef, useCallback } from "react";
-// import { invoke } from "@forge/bridge";
+import { invoke } from "@forge/bridge";
 import Canvas from "./Canvas";
 
 function App() {
   const [data, setData] = useState(null);
 
-  //   useEffect(() => {
-  //     invoke("getText", { example: "my-invoke-variable" }).then(setData);
-  //   }, []);
+  useEffect(() => {
+    invoke("getText", { example: "my-invoke-variable" }).then(setData);
+  }, []);
 
   return (
     <div>
-      {/* {data ? data : "Give me a minute..."} */}
+      {data ? data : "Give me a minute..."}
```

Comment:
I'm thinking once we know what apis we want, we can mock the data, so we can run the UI app locally using `npm run start` in our browser, instead of having to run `npm run build` each time, and checking changes on atlassian forge site.